Client: Updating: userdata-bbb_multi_user_tools, userdata-bbb_presenter_tools and userdata-bbb_multi_user_pen_only (#20218)
bigbluebutton/bigbluebutton: docs/docs/administration/customize.md

| Parameter                           | Description                                                                                                     | Default value |
| ----------------------------------- | --------------------------------------------------------------------------------------------------------------- | ------------- |
| `userdata-bbb_multi_user_pen_only=` | If set to `true`, only the pen tool will be available to non-participants when multi-user whiteboard is enabled | `false`       |

> | `userdata-bbb_presenter_tools=`     | Pass in an array of permitted tools from `settings.yml`. The options we support are: `select, hand, draw, eraser, arrow, text, note, rectangle and *more`. Example: "userdata-bbb_presenter_tools=['eraser', 'diamond']" will allow only the eraser and diamond tools.                                                     | all enabled   |

Reviewer: ```suggestion
| `userdata-bbb_presenter_tools=`     | Pass in an array of permitted tools from `settings.yml`. The options we support are: `select`, `hand`, `draw`, `eraser`, `arrow`, `text`, `note`, `rectangle` and *`more`. Example: `userdata-bbb_presenter_tools=['eraser', 'diamond']` will allow only the eraser and diamond tools.                                                     | all enabled   |
```

> | `userdata-bbb_presenter_tools=`     | Pass in an array of permitted tools from `settings.yml`. The options we support are: `select`, `hand`, `draw`, `eraser`, `arrow`, `text`, `note`, `rectangle` and *`more`. Example: `userdata-bbb_presenter_tools=['eraser', 'diamond']` will allow only the eraser and diamond tools.                                                     | all enabled   |

Reviewer: ```suggestion
| `userdata-bbb_presenter_tools=`     | Pass in an array of permitted tools from `settings.yml`. The options we support are: `select`, `hand`, `draw`, `eraser`, `arrow`, `text`, `note`, `rectangle` and *`more`. Example: `userdata-bbb_presenter_tools=['eraser', 'note']` will allow only the eraser and note tools.                                                     | all enabled   |
```

Reviewer: diamond is part of "more"

> | `userdata-bbb_multi_user_tools=`    | Pass in an array of permitted tools for non-presenters from `settings.yml`. The options we support are: `select, hand, draw, eraser, arrow, text, note, rectangle and *more`. Example: "userdata-bbb_multi_user_tools=['eraser', 'diamond']" will allow only the eraser and diamond tools.                                          | all enabled   |

Reviewer: ```suggestion
| `userdata-bbb_multi_user_tools=`    | Pass in an array of permitted tools for non-presenters from `settings.yml`. The options we support are: `select`, `hand`, `draw`, `eraser`, `arrow`, `text`, `note`, `rectangle` and *`more`. Example: `userdata-bbb_multi_user_tools=['eraser', 'diamond']` will allow only the eraser and diamond tools.                                          | all enabled   |
```

> | `userdata-bbb_multi_user_tools=`    | Pass in an array of permitted tools for non-presenters from `settings.yml`. The options we support are: `select`, `hand`, `draw`, `eraser`, `arrow`, `text`, `note`, `rectangle` and *`more`. Example: `userdata-bbb_multi_user_tools=['eraser', 'diamond']` will allow only the eraser and diamond tools.                                          | all enabled   |

Reviewer: ```suggestion
| `userdata-bbb_multi_user_tools=`    | Pass in an array of permitted tools for non-presenters from `settings.yml`. The options we support are: `select`, `hand`, `draw`, `eraser`, `arrow`, `text`, `note`, `rectangle` and *`more`. Example: `userdata-bbb_multi_user_tools=['eraser', 'note']` will allow only the eraser and note tools.                                          | all enabled   |
```

> *more: More includes the rest of the extra shapes, those being: `rectangle, ellipse, diamond, traingle, trapezoid, rhombus, hexagon, cloud, star, oval, x-box,check-box, arrow left, arrow up, arrow down, arrow right, frame, line, laser`.

Reviewer: ```suggestion
*more: More includes the rest of the extra shapes, those being: `rectangle`, `ellipse`, `diamond`, `traingle`, `trapezoid`, `rhombus`, `hexagon`, `cloud`, `star`, `oval`, `x-box`, `check-box`, `arrow left`, `arrow up`, `arrow down`, `arrow right`, `frame`, `line`, `laser`.

@Scroody what's the exact name of `arrow up`? Does it have underscore as a divider?
```

> *more: More includes the rest of the extra shapes, those being: `rectangle`, `ellipse`, `diamond`, `traingle`, `trapezoid`, `rhombus`, `hexagon`, `cloud`, `star`, `oval`, `x-box`, `check-box`, `arrow left`, `arrow up`, `arrow down`, `arrow right`, `frame`, `line`, `laser`.

Reviewer: ```suggestion
*more: More includes the rest of the extra shapes, those being: `rectangle`, `ellipse`, `diamond`, `triangle`, `trapezoid`, `rhombus`, `hexagon`, `cloud`, `star`, `oval`, `x-box`, `check-box`, `arrow-left`, `arrow-up`, `arrow-down`, `arrow-right`, `frame`, `line`, `laser`.
```

> OBS: Each tool/shape can be used separately by simply being passed in the parameter or setting by it's name. Example:  "user-data-bbb_example_paremeter=['triangle', 'cloud']".

Reviewer: ```suggestion
OBS: Each tool/shape can be used separately by simply being passed in the parameter or setting by it's name. Example:  `user-data-bbb_example_paremeter=['triangle', 'cloud']`.
```

> OBS: Each tool/shape can be used separately by simply being passed in the parameter or setting by it's name. Example:  `user-data-bbb_example_paremeter=['triangle', 'cloud']`.

Reviewer: ```suggestion
```
